src/pages/forgotPage/ForgotPage.jsx

```diff
@@ -21,70 +21,76 @@ const ForgotPassword = () => {
-
-        {/* Container for the main input fields */}
-        <div className="input-container">
-          {/* Header section with title and description */}
-          <div className="forgot-heading">
-            <h3>Forgot Password?</h3>
-            <p>No Worries we’ll send you reset instructions</p>
+        <div className="forgot-container flex">
+          <div className="login-image-container">
+            <img src={forgotPasswordImage} alt="ForgotPasswordImage" />
           </div>
+          <div className="">
+            <div className="forgot-form-container">
+             
+                <a className="back" href="/">
+                  <span>
+                    <img src="/icons/backIcon.svg" alt="" />
+                  </span>
+                  <p className="backtologin">Back to login</p>
+                </a>
+             
+              <div className="form-heading">
+                <h1>Forgot Password?</h1>
+                <p>No Worries we’ll send you reset instructions</p>
+              </div>
 
-          <div className="form-container">
-            <form action="" onSubmit={handelSubmit}>
-              {/* Email input field section */}
-              <div className="email-input">
-                <label htmlFor="Email">
-                  Email
-                  <span className="required-symbole">*</span>
-                </label>
-                <div className="emailAndicon">
-                  <span className="material-symbols-outlined">mail</span>
-                  <input
-                    autoComplete="off"
-                    type="text"
-                    className="forgot-password-input"
-                    placeholder="Input Email"
-                    value={email}
-                    onChange={handleEmail}
-                    name="email"
-                  />
-                </div>
+              <div className="form-container">
+                <form action="" onSubmit={handelSubmit}>
+                  {/* Email input field section */}
+                  <div className="email-container flex">
+                    <label htmlFor="Email">
+                      Email Id
+                      <span className="required-symbole">*</span>
+                    </label>
+                    <div className="">
+                      {/* <span className="material-symbols-outlined">mail</span> */}
+                      <input
+                        autoComplete="off"
+                        type="text"
+                        className="email-input"
+                        placeholder="Type here..."
+                        value={email}
+                        onChange={handleEmail}
+                        name="email"
+                      />
+                    </div>
 
-                <p className="error-message">
-                  {emailError.email ? emailError.email : ""}
-                </p>
-              </div>
+                    <p className="error-message">
```

Comment:
Email  Field is required - Required field cannot be empty.
Email not valid  - Invalid Email

Reply:
Changed. Email error messages